sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -329,12 +261,15 @@ void receiveTwoMessagesAutoComplete(MessagingEntityType entityType, boolean isSe
         StepVerifier.create(receiver.receiveMessages())
             .assertNext(receivedMessage -> {
                 assertMessageEquals(receivedMessage, messageId, isSessionEnabled);
+                receiver.complete(receivedMessage).block(Duration.ofSeconds(15));
```

Comment:
Why are we explicitly completing these? the method suggests that these should be auto completed `receiveTwoMessagesAutoComplete`. Same with the one below.